athena-dynamodb/README.md

```diff
@@ -51,14 +51,19 @@ Review the "Policies" section of the athena-dynamodb.yaml file for full details
 4. CloudWatch Logs - This is a somewhat implicit permission when deploying a Lambda function but it needs access to cloudwatch logs for storing logs.
 1. Athena GetQueryExecution - The connector uses this access to fast-fail when the upstream Athena query has terminated.
 
+### Running Integration Tests
+
+See [Running Integration Tests](https://github.com/awslabs/aws-athena-query-federation/blob/master/athena-federation-integ-test/README.md#running-integration-tests).
+
 ### Deploying The Connector
 
 To use this connector in your queries, navigate to AWS Serverless Application Repository and deploy a pre-built version of this connector. Alternatively, you can build and deploy this connector from
 source follow the below steps or use the more detailed tutorial in the athena-example module:
 
-1. From the athena-federation-sdk dir, run `mvn clean install` if you haven't already.
-2. From the athena-dynamodb dir, run `mvn clean install`.
-3. From the athena-dynamodb dir, run  `../tools/publish.sh S3_BUCKET_NAME athena-dynamodb` to publish the connector to your private AWS Serverless Application Repository. The S3_BUCKET in the command
+1. From the **athena-federation-sdk** dir, run `mvn clean install` if you haven't already.
+2. From the **athena-federation-integ-test** dir, run `mvn clean install` if you haven't already.
+3. From the **athena-dynamodb** dir, run `mvn clean install`.
```

Comment:
Will this run integration tests? hopefully not since mvn clean install should be able to succeed in a sandbox (e.g. no remote resources)

Reply:
No, only UT. To run integration tests you need to explicitly run: `mvn failsafe:integration-test`.

Reply:
so why do these instructions say to build the integ test package? should we caveat this to say (only if you also plan to run mvn failsafe:integration-test ?

Reply:
Failure to compile the module first will result in a compilation error (added note).